features/upgrade/sdn/policy-upgrade.feature

```diff
@@ -0,0 +1,29 @@
+Feature: SDN compoment upgrade testing
+
+  # @author huirwang@redhat.com
+  @admin
+  @upgrade-prepare
+  Scenario: network operator should be available after upgrade - prepare
+  # According to our upgrade workflow, we need an upgrade-prepare and upgrade-check for each scenario.
+  # But some of them do not need any prepare steps, which lead to errors "can not find scenarios" in the log.
+  # So we just add a simple/useless step here to get rid of the errors in the log.
+    Given the expression should be true> "True" == "True"
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-22707
+  @admin
+  @upgrade-check
+  Scenario: network operator should be available after upgrade
+    Given I switch to cluster admin pseudo user
+    When I use the "openshift-network-operator" project
+    Then status becomes :running of exactly 1 pods labeled:
+      | name=network-operator |
+    # Check network operator version match cluster version
+    And the "network" operator version matchs the current cluster version
```

Comment:
typo in the step defintion itself.  Once #1026 is merged, please change from `matchs` to `matches`.  LGTM otherwise.

Reply:
@huiran0826 #1026 has been merged, can you pls update the step accordingly?

Reply:
@pruan-rht Thanks peter, fixed it.